Extensions/tf2_reblogs.js

```diff
@@ -75,7 +75,7 @@ XKit.extensions.tf2_reblogs = new Object({
 			$(this).addClass("tf2_done");
 			var iconurl = icons[$(this).closest("[data-id]").attr("data-id") % icons.length];
```

Comment:
this can't possibly be handling the BigInt conversion correctly

Reply:
Yyyyyep, definitely isn't.